collectors/charts.d.plugin/speedtest/speedtest.chart.sh

```diff
@@ -43,9 +43,9 @@ speedtest_get() {
   # 4. USE LOCAL VARIABLES (global variables may overlap with other modules)
 
   output=$(speedtest --format=csv)
-  
-  speedtest_download=$(echo "$output" | awk -F, '{print $6}' | tr -d '"')
-  speedtest_upload=$(echo "$output" | awk -F, '{print $7}' | tr -d '"')
+
+  speedtest_download=$(($(echo "$output" | awk -F, '{print $6}' | tr -d '"')*8/1024))
+  speedtest_upload=$(($(echo "$output" | awk -F, '{print $7}' | tr -d '"')*8/1024))
```

Comment:
This is kilobytes, not kilobits

Reply:
yes - turns out that speedtest csv format always returns data in bytes, so I was converting into bits and then dividing. 

we use 1000 not 1024?

Reply:
for bits 1000, yes.

bytes => kilobits: X / 8 / 1000.

Reply:
@ilyam8 

Example: 

speedtest_download == 11437598 bytes/s

speedtest_download /8/1000 == 1429 kilobits per second  (This is wrong)

speedtest_download *8/1000 == 91500 kilobits per second (This is correct)

Hence, X * (8 / 1000) - what am I missing here 😄 

(btw, updated 1024 to 1000)

Reply:
correct, it is `*` (8 bits in a byte).